README.md

```diff
@@ -344,6 +356,33 @@ vars:
 * `variable` (required): The variable containing the calculated field `name` and `transform_sql`.
 
 ----
+
+### drop_schemas_automation ([source](macros/drop_schemas_automation.sql))
+This macro was created to be clean up the schemas in our integration test environments. It drops schemas that are `like` the `target.schema`. By default it will drop the target schema as well but this can be configured.
```

Comment:
```suggestion
This macro was created to help clean up the schemas in our integration test environments. It drops schemas that are `like` the `target.schema`. By default it will drop the target schema as well but this can be configured.
```